coops-ui/tests/unit/OfficeAddresses.spec.ts

```diff
@@ -0,0 +1,411 @@
+import Vue from 'vue'
+import Vuelidate from 'vuelidate'
+import Vuetify from 'vuetify'
+import store from '@/store/store'
+
+import { OfficeAddresses } from '@/components/Common'
+import { mount, Wrapper } from '@vue/test-utils'
+import { EntityTypes } from '@/enums'
+
+Vue.use(Vuetify)
+Vue.use(Vuelidate)
+
+let vuetify = new Vuetify({})
+
+// Boilerplate to prevent the complaint "[Vuetify] Unable to locate target [data-app]"
+const app: HTMLDivElement = document.createElement('div')
+app.setAttribute('data-app', 'true')
+document.body.append(app)
+
+describe('OfficeAddresses as a COOP', () => {
+  let vm
+
+  beforeAll(() => {
+    // init store
+    store.state.entityType = EntityTypes.Coop
+    store.state.registeredAddress = {
+      deliveryAddress: {
+        addressCity: 'delCity',
+        addressCountry: 'delCountry',
+        addressRegion: 'delRegion',
+        deliveryInstructions: 'delInstructions',
+        postalCode: 'delPostal',
+        streetAddress: 'delStreet',
+        streetAddressAdditional: 'delStreetAdd'
+      },
+      mailingAddress: {
+        addressCity: 'mailCity',
+        addressCountry: 'mailCountry',
+        addressRegion: 'mailRegion',
+        deliveryInstructions: 'mailInstructions',
+        postalCode: 'mailPostal',
+        streetAddress: 'mailStreet',
+        streetAddressAdditional: 'mailStreetAdd'
+      }
+    }
+  })
+
+  const draftAddresses = {
+    registeredOffice: {
+      deliveryAddress: {
+        addressCity: 'delCityDraft',
+        addressCountry: 'delCountryDraft',
+        addressRegion: 'delRegionDraft',
+        deliveryInstructions: 'delInstructionsDraft',
+        postalCode: 'delPostalDraft',
+        streetAddress: 'delStreetDraft',
+        streetAddressAdditional: 'delStreetAddDraft'
+      },
+      mailingAddress: {
+        addressCity: 'mailCityDraft',
+        addressCountry: 'mailCountryDraft',
+        addressRegion: 'mailRegionDraft',
+        deliveryInstructions: 'mailInstructionsDraft',
+        postalCode: 'mailPostalDraft',
+        streetAddress: 'mailStreetDraft',
+        streetAddressAdditional: 'mailStreetAddDraft'
+      }
+    }
+  }
+
+  it('loads the current office addresses properly', async done => {
```

Comment:
Async is not needed since nothing in here awaits anything. I think this applies to all tests in this file.

Reply:
Correct indeed, there is nothing being awaited. Removing!